tests/test_math.py

```diff
@@ -47,6 +47,20 @@ def test_floor(self):
                                                                     [7., 8.]])))
 
 
+class tanhTests(unittest.TestCase):
+    def test_tanh(self):
+        # int
+        t1 = TensorBase(np.array([[-0, 1, -2], [0, -1, 2]]))
+        t2 = syft.math.tanh(t1)
+        self.assertTrue(np.array_equal(t1.data, np.array([[0, 1, -2], [0, -1, 2]])))
+        self.assertTrue(np.array_equal(t2.data, np.array([[0, np.tanh(1), np.tanh(-2)], [0, np.tanh(-1), np.tanh(2)]])))
+        # float
+        t1 = TensorBase(np.array([[-0.0, 1.5, -2.5], [0.0, -1.5, 2.5]]))
+        t2 = syft.math.tanh(t1)
+        self.assertTrue(np.array_equal(t1.data, np.array([[0.0, 1.5, -2.5], [0.0, -1.5, 2.5]])))
+        self.assertTrue(np.array_equal(t2.data, np.array([[0.0, np.tanh(1.5), np.tanh(-2.5)], [0.0, np.tanh(-1.5), np.tanh(2.5)]])))
```

Comment:
why not this ? : `np.tanh(np.array([[0, 1, -2], [0, -1, 2]]))`

Reply:
and `np.tanh(np.array([[0.0, 1.5, -2.5], [0.0, -1.5, 2.5]]))`

Reply:
Hmm I'm guessing that I was going for slightly less abstraction. I don't feel strongly about it, do you want me to change it?

Reply:
yes.